Shape_detection/include/CGAL/Shape_detection/Efficient_RANSAC/Efficient_RANSAC.h

```diff
@@ -18,6 +18,8 @@
 
 #include <CGAL/Random.h>
 
+#include <CGAL/Point_set_3.h>
```

Comment:
Is a forward declaration enough?  This `#include` needs changes in the file `dependencies`